### PR TITLE
BBBSWA-38 Creating the footer component

### PR DESCRIPTION
**Pull request by velvetarchangel**
Created the footer component so it is reuseable for every page we will
use. Also created different vues for applicant administrator and
Admindashboard.

### CODE REVIEW

**Comment by ckthiessen** [comment]
The text on the right is not in-line with the text on the left. It should be vertically centered in the div like the buttons are. 
image

File: frontEnd/src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      </v-btn>
+    </v-col>
+    <v-col
+        text
+        font-family='Roboto, sans-serif'
+        rounded
+        align="right"
+        class="primary"
+      >
+      © 2020 - Big Brothers Big Sisters Calgary
```
Comment (ckthiessen) [inline frontEnd/src/components/Footer.vue]: Try <span class-"my-auto"> © 2020 - Big Brothers Big Sisters Calgary </ span>

Comment (velvetarchangel) [reply frontEnd/src/components/Footer.vue]: did not seem to do the trick sadly :(. Did the best I can for now if it is an issue for the client will fix later.

Comment (ckthiessen) [reply frontEnd/src/components/Footer.vue]: style="margin: auto 0 auto 0" should work instead. "my-auto" is just a class with that style (but I can't get the vuetify CSS library working).

Comment (ckthiessen) [reply frontEnd/src/components/Footer.vue]: my-auto stands for margin-y-axis-auto. There is a similar one for centering stuff horizontally (mx-auto)

**Pull request merged**